### ISSUE TITLE
Update for Moodle 4.0

### ISSUE THREAD

**Issue by FahadKhanChughtai** [bmbrands/moodle-local_welcome#18]
Hi? 

Is there any way to install this in the latest version of Moodle? When is it getting updated?

**Comment by ecottis** [comment]
I'm also interested in when this will be adapted/updated for Moodle 4.1 as its a really useful plugin

**Comment by rajib4** [comment]
Yup, it would be great if it is updated to latest Moodle v4.x

**Comment by ecottis** [comment]
I agree a great plug-in that informs students of their details saving a lot of time in having to send out additional welcome messages. Should ideally be integrated into the moodle core.
